README.md

```diff
@@ -4,27 +4,43 @@ Automating MakePlayingCards's online ordering system.
 The below guide describes the procedure for setting up the web component. If you're here to download the clientside program, check the Releases tab.
 
 # Requirements
-`requirements.txt` for the web application and local tool combined exists in the repo as well.
+`requirements.txt` for the web application and local tool combined.
 
-## Web application:
-JS libraries:
-* `bootstrap@4.6.0`
-* `jquery-ui-touch-punch@0.2.3`
-* `slick.js@1.8.1`
-
-Other:
-* [Bootstrap Superhero](https://bootswatch.com/superhero/)
+* Python3.8+
 * Elasticsearch (probably easiest with Docker)
 * A Google account
 
 # Setup
-1. Clone this repo somewhere on your server
-2. In the same directory as the repo, create a folder called `staticroot` for static assets
-3. Deploy the Django project (I'm using DigitalOcean for Ubuntu) with a webserver (I'm using Apache) and serve static files with another webserver if you want (I was previously using nginx but now I just serve static files with Apache as well)
-4. Run Elasticsearch
-5. Set up your Google Drive credentials in the `MPCAutofill` directory (base Django directory). You should set up a Google Drive service account and store your credentials as `client_secrets.json`
-6. Run the command `manage.py import_sources` to sync sources in `drives.csv` to database, and `manage.py update_database` to populate the database (optionally specifying a particular drive to sync with `-d <drivename>`)
-7. Create a cronjob to periodically run the database updater command, to ensure MPC Autofill reflects the current state of the linked Drives, and another cronjob to periodically synchronise the double-faced cards table with Scryfall:
-* `0 0 * * * bash /root/mpc-autofill/update_database >> /root/db_update.txt 2>&1`
-* `0 0 * * SUN bash /root/mpc-autofill/sync_dfcs`
-8. Deploy two Google Script according to the code specified in `autofill.py` and adjust the URLs in that script to point to your GS endpoints
+#### <em>All steps with the prefix [PROD] are only needed if you are deploying this web app in a production environment. For running the application locally or for development, you can skip these steps.</em>
+1. Clone this repo somewhere on your server/local computer. This will be referred to as the `$working_directory`.
+2. If you are using an IDE to run the Django commands and are NOT using Windows: go to the `$working_directory/.run` directory and for each XML file delete the option `SDK_HOME`
+3. Add the file `$working_directory/MPCAutofill/MPCAutofill/.env` to edit any environment variables used in `$working_directory/MPCAutofill/MPCAutofill/settings.py`.  
+   IN ORDER TO RUN THE APP LOCALLY, YOU MUST ADD `DJANGO_DEBUG=on` TO THE `$working_directory/MPCAutofill/MPCAutofill/.env` FILE
```

Comment:
might make more sense to make `DEBUG` in settings default to True to remove this step?

Reply:
That might be best, however this step does direct the user's attention to the usage of the git-ignored dotenv file. 

I'm actually not 100% sure it needs to be in DEBUG mode to run locally. It would only run for me when I changed it, but @fklemme's fork that incorporates Docker has DEBUG off. Probably because his Docker version creates the static files or something. I'm not sure what the best choice is.

Reply:
I was also surprised to read that DEBUG=True is required in your setup!? I only tried it with Docker and both seems to work fine with me. So what is the problem when turning DEBUG off?

Reply:
what happened when you tried to run it locally with debug=False?

Reply:
I just get a SERVER ERROR (500) on initial page load.

Reply:
This might have something to do with never creating the static files.